### PR TITLE
Moved copy tasks from date to the actions box

### PR DESCRIPTION
**Pull request by pabelenda**
This way, we are saving more vertical space.

Changes:

 * Added a new section at the actions box
 * Changed layout into an hbox
 * Adjusted width and margins of both elements
 * Shortened button text

### CODE REVIEW

**Comment by jaragunde** [comment]
Aside from the changes requested, I think it looks good and we have recovered all the space we had lost with the new, extended work summary panel.

File: web/js/tasks.js [hunk 1]
```diff
@@ -1049,29 +1049,26 @@ Ext.onReady(function(){
...
                 allowBlank: false,
                 startDay: 1,
             }, new Ext.Button({
-                text:'Copy tasks from selected date',
-                width: 60,
-                margins: "7px 0 0 0px",
+                text:'Copy',
+                width: 96,
+                margins: "0px 0 0 0px",
+                flex: 2,
```
[the remark this answers is not in the document]
Comment (pabelenda) [reply web/js/tasks.js]: Actually mixing both `width` and `flex` was causing some "funny" effects.

I've removed `width` as suggested and changed into `flex: 3` and `flex: 1`

File: web/js/tasks.js [hunk 2]
```diff
@@ -1049,29 +1049,26 @@ Ext.onReady(function(){
     // Cloning Panel
     var cloningPanel = new Ext.FormPanel({
         width: 204,
-        height: 65,
         renderTo: Ext.get('calendarpanel'),
```
[the remark this answers is not in the document]
Comment (pabelenda) [reply web/js/tasks.js]: This was the first thing I knew I had to change but somehow forgot. And it worked!

File: web/js/tasks.js [hunk 3]
```diff
@@ -1049,29 +1049,26 @@ Ext.onReady(function(){
     // Cloning Panel
     var cloningPanel = new Ext.FormPanel({
         width: 204,
```
[the remark this answers is not in the document]
Comment (pabelenda) [reply web/js/tasks.js]: +1000 to have less hard coded sizes.

**Comment by pabelenda** [comment]
Changes requested should be on place.

**Comment by jaragunde** [comment]
This is looking great, thank you. What do you think of the patch I just added? I think it makes the actions panel feel less crowded.

**Comment by pabelenda** [comment]
IMO it looks better with this last changes. Merge! Merge! Merge!

**Pull request merged**